### ISSUE TITLE
Filter Deprecation warnings during simulation

### ISSUE THREAD

**Issue by akouete-kpakpo** [ML-for-B-E/nevergrad#20]
## Context
Deprecation warnings pollute the log during the execution of an experiment

## Solution proposed
Add the line 
```python
import warnings
warnings.filterwarnings("ignore", category=DeprecationWarning)
```
in `nevergrad.benchmark.__main__.py`

**Comment by akouete-kpakpo** [comment]
FYI @chrichri17

**Comment by chrichri17** [comment]
@akouete-kpakpo I tried this already and it didn't seem to be working.

**Comment by akouete-kpakpo** [comment]
Closed via #23
